submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,101 @@
+
+{% include gallery.html items='RLD-GUI.png;Rail Logistics Dispatcher's GUI,LTN-Combinator-Modernized.png;LTN Combinator Modernized's GUI' %}
+
+While third party mods like [LTN Combinator Modernized](https://mods.factorio.com/mod/LTN_Combinator_Modernized) have been very helpful to those lacking experience with circuit network, the fact remains that it is an entirely separate mod dedicated to simplfying and improving the user experience. With that in mind, RLD opted to include a GUI right out of the box. (RLD does also offer a 'signals mode' to allow the user to configure the Dispatcher in a manner more similar to LTN if desired.) Similar to Project Cybersyn above, Rail Logistics Dispatcher's author also plans to implement a native [manager UI](https://mods.factorio.com/mod/RailLogisticsDispatcher/discussion/6390043c3a48c93b86a5f22b) to view the whole network in the future.
+
+Unlike both of the previous two mods, Rail Logistics Dispatcher adds no new signals at all. Everything that was previously done through signals such as Thresholds, Train Configurations, Network IDs, is instead done through use of the GUI. RLD also goes all in on the the per-item thresholds. It doesn't have any per-station thresholds through its GUI, nor any global thresholds in its settings. However, it can save certain requests and thresholds to be applied again later. In addition to the individual item units and stacks of items, RLD also has the ability to set requests based on a wagon (fluid or cargo) load.
+
+Instead of the "Locked Slots Signal" that LTN and CYbersyn share, RLD instead opts to reverse inserters and have them put items back inside the chest or warehouse from which they picked it up. On one hand, this does mean that the user would have difficulty making stations using loaders or inserters picking up directly from belts. On the other, this does mean that multi-item provider trains can fill trains with slightly higher loads.
+
+Rail Logistics Dispatcher's GUI makes it the easiest of the three mods to use, but it does come with one big downside: While copying a dispatcher within the same save is perfectly doable, blueprinting it for use in other saves or for sharing with others is not. API limitations mean that only the Dispatcher entity is copied, while the configured settings are lost. A second, more minor issue is that you may need to isolate the signals from your inventory with a combinator, or use a different wire, if you wish to use them for other purposes, due RLD's Dispatcher input having negative signals for items it is requesting that might otherwise contaminate it.
```

Comment:
```suggestion
Rail Logistics Dispatcher's GUI makes it the easiest of the three mods to use, but it does come with one big downside: while copying a dispatcher within the same save is perfectly doable, blueprinting it for use in other saves or for sharing with others is not. API limitations mean that only the Dispatcher entity is copied, while the configured settings are lost. A second, more minor issue is that you may need to isolate the signals from your inventory with a combinator, or use a different wire, if you wish to use them for other purposes, due to RLD's Dispatcher input having negative signals for items it is requesting that might otherwise contaminate it.
```